README.md

```diff
@@ -1 +1,92 @@
-# cumulus-message-adapter-js
\ No newline at end of file
+# @cumulus/cumulus-message-adapter-js
```

Comment:
Maybe add something about how this needs to be packaged for lambda

Reply:
It seems like teaching people how to create a Lambda package is out of scope for this, which is just about how to use the message adapter.  If there's some wiki page on how to do that then I can link to it.  @scisco, is there any page like that out there?

Reply:
Talked to @scisco earlier today and he said to put it in the README. But up to you. I am going to work on a better format for this.

Reply:
Fixed.